### DISCUSSION TITLE
practice-exam/practice-exam-3

### DISCUSSION THREAD

**Discussion by giscus** [kananinirav/AWS-Certified-Cloud-Practitioner-Notes#78] [Comments]
# practice-exam/practice-exam-3

If you are preparing for AWS Cloud Practitioner Exam, this guide will help you with quick revision before the exam. I share the notes I used to study and pass exam.

https://kananinirav.com/practice-exam/practice-exam-3.html

**Comment by luca-rui** [comment]
> A company needs to host a database in Amazon RDS for at least three years. Which of the following options would be the most cost-effective solution?
>
>    A. Reserved instances - No Upfront.
>    B. Reserved instances - Partial Upfront.
>    C. On-Demand instances.
>    D. Spot Instances.

The right answer is listed as B, but I believe it should be D, seen as is the most **cost-effective** solution and there is no indication in the question that availability is a priority.

**Reply by MichielVrins** [reply]
Spot instances are a pricing feature of EC2, not of Amazon RDS. So the answer should indeed be B.

**Reply by anasulayeu** [reply]
What is the most cost-effective purchasing option for running a set of EC2 instances that must always be available for a period of two months?

    A. On-Demand Instances.
    B. Spot Instances.
    C. Reserved Instances - All Upfront.
    D. Reserved Instances - No Upfront.

Correct answer: C

Reserved Instances (RIs) offer a significant discount compared to On-Demand Instances, and they provide a capacity reservation for the specified instance type in a specific region. When you choose the "All Upfront" payment option for Reserved Instances, you pay for the entire term upfront, and in return, you receive the highest discount.

This payment option provides the maximum cost savings over the term of the reservation. However, keep in mind that with this payment model, you pay the full cost at the beginning of the reservation period.

**Reply by holopekochan** [reply]
You could lost your instance and data for spot instance. Not suitable for database usage.
